apps/demo-game/src/pages/play/cockpit.tsx

```diff
@@ -54,7 +87,74 @@ function GameLayout({ children }: { children: React.ReactNode }) {
     },
   }
 
-  const sidebar = <div>SidebarAddons</div>
+  const sidebar = (
+    <div className="flex items-center justify-between">
+      <Switch
+        className={{
+          root: 'text-xs font-bold text-gray-600',
+        }}
+        disabled={!data.self || loading}
+        id="isReady"
+        checked={data.self.isReady}
+        label="Ready?"
+        onCheckedChange={async () => {
+          await updateReadyState({
+            variables: {
+              isReady: !data.self.isReady,
+            },
+          })
+        }}
+      />
+
+      {countdownDurationMs !== null && (
+        <CycleCountdown
+          className={{
+            root: '',
+            countdownWrapper: '',
+            countdown: 'text-xs font-bold text-gray-600',
+          }}
+          totalDuration={countdownDurationMs / 1000}
+          expiresAt={new Date(strExpiresAt)}
```

Comment:
we almost never use Date directly as it has so many quirks and behaves weirdly. we have very good experiences with dayjs for actual date handling, and date-fns is nice too when just computing stuff based on dates (as you have used already). so here would just do dayjs(strExpiresAt)

Reply:
I see the motivation behind using dayjs. But the CycleCountdown is expecting a date, that's why I used Date ^^. Let me know if I should change that in the design system!

For the other use-cases I will check what I can do.

Reply:
Please double-check my changes - I could replace all the Date's with dayjs

For the CycleCountdown I just do `dayjs(strExpiresAt).toDate()`